Source/Scripts/cheat_core_player.lua

```diff
@@ -260,45 +428,48 @@ cheat.cheat_tp_tr_args = {
 }
 
 cheat:createCommand("cheat_tp_tr", "cheat:tp_tr(%line)", cheat.cheat_tp_tr_args,
-"Teleports the player to the given place. Supported places (case insensitive):\n$8I, II, III, IV, V, VI, VII, VIII, IX, X, XI, XII, XIII, XIV, XV, XVI, XVII, XVIII, XIX, XX,\n$8XXI, XXII, XIII, XXIV, XXV, AI, AII, AII, AIII, AIV, AV",
-"Example", "cheat_tp_tr place:XXV")
+"Teleports the player to the given place. Supported places (case insensitive):\n$8TR_1, TR_2, TR_3, TR_4, TR_5, TR_6, TR_7, TR_8, TR_9, TR_10, TR_11, TR_12, TR_13, TR_14, TR_15,\n$8TR_16, TR_17, TR_18, TR_19, TR_20, TR_21, TR_22, TR_23, TR_24, TR_25, ATR_1, ATR_2, ATR_3, ATR_4, ATR_5",
+"Example", "cheat_tp_tr place:TR_1")
 function cheat:tp_tr(line)
   local gender = player.soul:GetGender()
   local args = cheat:argsProcess(line, cheat.cheat_tp_tr_args)
   local nplace, nplaceErr = cheat:argsGet(args, 'place')
-
+  
+  -- All treasure locations. I preferred arabic numerals to roman
+  -- numerals because it is much easier to visit all of them in order.
+  -- (You normally only need to change one digit and re-run the command)
```

Comment:
I think removing the prefix would make this faster to type. Also why TR_ and ATR_ ?

Reply:
Sure, it would be faster to type but we have normal treasure maps (TR) and ancient treasure maps (ATR). The counting for both starts by one. I can remove the "R" from both to simplify the typing or create a separate command for the ancient treasures and stick only with numbers.